docs/en/transform-v2/explode.md

```diff
@@ -0,0 +1,67 @@
+# Explode
+
+> Explode transform plugin
+
+## Description
+
+Split a row of data into multiple data according to the specified delimiter.
+
+## Options
+
+|      name      | type | required | Description   |
+|----------------|------|----------|---------------|
+| explode_fields | Map  | yes      | Explode field |
+
+
+### explode_fields [Map]
+
+The fields that need to be separated by the specified delimiter
+The map is composed of field as key and separator as value.
+
+### common options [string]
+
+Transform plugin common parameters, please refer to [Transform Plugin](common-options.md) for details
+
+## Example
+
+The data read from source is a table like this:
+
+| name              | age | card    |
+|-------------------|-----|---------|
+| Joy Ding,May Ding | 20  | 123;234 |
+| Kin Dom,Joy Dom   | 20  | 123;345 |
+
+we want to explode name and card, we can add a `Explode` Transform like below:
+
+```
+transform {
+  Explode {
+    source_table_name = "fake"
+    result_table_name = "fake1"
+    explode_fields = {"name":",","card":";"}
```

Comment:
I think we should support list type value too. For example, 
| name              | age | value list    |
|-------------------|-----|---------|
| Joy Ding,May Ding | 20  | [123, 234] |
| Kin Dom,Joy Dom   | 20  | [123, 345] |

Reply:
> I think we should support list type value too. For example, 
> 
> | name              | age | value list    |
> 
> |-------------------|-----|---------|
> 
> | Joy Ding,May Ding | 20  | [123, 234] |
> 
> | Kin Dom,Joy Dom   | 20  | [123, 345] |

Nice idea, maybe only for values due to we can lose link between column name and values if we use list type for column names

Reply:
> I think we should support list type value too. For example, 
> 
> | name              | age | value list    |
> 
> |-------------------|-----|---------|
> 
> | Joy Ding,May Ding | 20  | [123, 234] |
> 
> | Kin Dom,Joy Dom   | 20  | [123, 345] |

Just don't understand how names are grouped. List type for values great idea but if we lose link between column name and values not so good.

Reply:
These are example. https://github.com/apache/seatunnel/pull/7928/files#diff-d4ae2963bb7289d64f3e406b727c6acf6081e9fb060c7ddbc4cfbbaa870822ceR25